src/components/OwnerPanel/OwnerPanel.tsx

```diff
@@ -170,6 +170,13 @@ const OwnerPanel = (props: Props) => {
         </div>
       </div> */}
 
+      <div className={styles.dotContainer}>
```

Comment:
Ah these dots are social share which are commented out below. We can remove these dots until we implement social share

Reply:
LOL I was wondering what the purpose of the dots were...that explains it